languages/php/menhir/parse_php.ml

```diff
@@ -256,12 +257,7 @@ let tmp_php_file_from_string ?(header = "<?php\n") s =
 
 (* this function is useful mostly for our unit tests *)
 let (tokens_of_string : string -> Parser_php.token list) =
- fun s ->
-  let tmpfile = Common.new_temp_file "pfff_tokens_of_s" "php" in
-  Common.write_file tmpfile ("<?php \n" ^ s ^ "\n");
-  let toks = tokens tmpfile in
-  Common.erase_this_temp_file tmpfile;
-  toks
+ fun str -> tokens (Parsing_helpers.Str str)
```

Comment:
hmm I'm surprised the tests are passing, but I think you'll need to also add the leading
<?php to str before calling tokens.